Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml

```
@@ -3,2308 +3,2174 @@ commonfields:
-    type: 15
-    additionalinfo: null
-    options:
-      - "2.0"
-      - "2.1"
-  - display: Fetch incidents
-    name: isFetch
-    required: false
-    type: 8
-  - display: Incident type
-    name: incidentType
-    required: false
-    type: 13
-  - name: fetch_time
-    display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
-      months, 1 year)
-    required: false
-    defaultvalue: 3 days
-    type: 0
-    additionalinfo: null
-  - name: fetch_threat_rank
-    display: Minimum risk score for importing incidents (0-10), where 0 is low risk
-      and 10 is high risk. Relevant for API version 2.0.
-    required: false
-    defaultvalue: "5"
-    type: 0
-    additionalinfo: null
-  - name: fetch_limit
-    display: "Fetch limit: The maximum number of incidents to fetch"
-    required: false
-    defaultvalue: "10"
-    type: 0
-    additionalinfo: null
-  - name: fetch_site_ids
-    display: Site IDs
-    required: false
-    defaultvalue: null
-    type: 0
-    additionalinfo: Comma-separated list of site IDs to fetch incidents for. Leave
-      blank to fetch all sites.
-  - name: block_site_ids
-    display: Block Site IDs
-    required: false
-    defaultvalue: null
-    type: 0
-    additionalinfo: Comma-separated list of site IDs for where hashes should be
-      blocked. If left blank all hashes will be blocked globally. If filled out
-      with site ids all hashes will be no longer be blocked globally, they will
-      now be blocked in the scope of those sites.
-  - name: insecure
-    display: Trust any certificate (not secure)
-    required: false
-    type: 8
-    additionalinfo: null
-  - name: proxy
-    display: Use system proxy settings
-    required: false
-    type: 8
-    additionalinfo: null
-  - name: token
-    display: API Token (Deprecated)
-    required: false
-    defaultvalue: null
-    type: 4
-    additionalinfo: Use the "API Token (Recommended)" parameter instead.
-  - name: incidentFetchInterval
-    display: Incidents Fetch Interval
-    required: false
-    defaultvalue: "1"
-    type: 19
-    additionalinfo: null
-description: Use the SentinelOne integration to send requests to your management
-  server and get responses with data pulled from agents or from the management
-  database.
+- name: url
+  display: Server URL (e.g., https://usea1.sentinelone.net)
+  required: true
+  type: 0
+- name: credentials
+  display: ""
+  required: false
+  type: 9
+  displaypassword: API Token
+  hiddenusername: true
+- name: api_version
+  display: API Version
+  required: true
+  defaultvalue: "2.1"
+  type: 15
+  options:
+  - "2.0"
+  - "2.1"
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- name: fetch_type
+  display: Defines if Alerts and/or Threat should be fetched
+  required: false
+  defaultvalue: Threats
+  type: 16
+  options:
+  - Alerts
+  - Threats
```

change to single select
```suggestion
- name: fetch_type
  display: Fetch incidents from type
  required: false
  defaultvalue: Threats
  type: 15
  options:
  - Alerts
  - Threats
  - Both
```